modules/thermal_hydraulics/test/tests/materials/ad_wall_heat_transfer_coefficient_kazimi/tests

```diff
@@ -0,0 +1,52 @@
+[Tests]
+  design = 'ADWallHeatTransferCoefficientKazimiMaterial.md'
+  issues = '#24340'
+  [normal_operation]
+    requirement = "The system shall compute the heat conductance using the Kazime-Carelli "
+                  "correlation for the Nusselt number"
+    type = 'CSVDiff'
+    input = 'kazimi_test.i'
+    csvdiff = 'kazimi_test_out.csv'
+  []
+  [abnormal_operation]
```

Comment:
Very thorough with the 4 tests, but let's just do 2: one for Pe out of bounds and one for P/D out of bounds. Do similarly for other materials.

Reply:
done